src/main/java/org/usfirst/frc4904/robot/humaninterface/operators/DefaultOperator.java

```diff
@@ -14,5 +22,13 @@ public DefaultOperator(String name) {
 	}
 
 	@Override
-	public void bindCommands() {}
+	public void bindCommands() {
+		RobotMap.HumanInput.Operator.joystick.button3.onlyWhileHeld(new HatchIntake());
+		RobotMap.HumanInput.Operator.joystick.button5.whenPressed(new VelcroPistonExtend());
+		RobotMap.HumanInput.Operator.joystick.button5.whenReleased(new VelcroPistonRetract());
+		RobotMap.HumanInput.Operator.joystick.button2
+			.onlyWhileHeld(new RunIfElse(new WristUp(), new WristDown(), RobotMap.Component.floorio.wrist::isExtended));
```

Comment:
wristUp and wristDown are instantaneous, so it doesn't make sense to put them in a whileHeld (whenPressed would do the same job better)